components/fxa-client/ios/FxAClient/FxAccountManager.swift

```diff
@@ -131,6 +139,38 @@ open class FxaAccountManager {
         }
     }
 
+    /// Use the provided user account information to sign-in without any user intervention.
+    public func migrationAuthentication(
```

Comment:
It's a nit, but, this name doesn't seem right to me - it's a "doing" method that is named with a noun rather than a verb, and the naming is not consistent with its partner method `retryMigration`. Would something like `authenticateViaMigration` work here? It feels like it would pair better with `retryMigration` in the caller's mental model of this API surface.